AutocompleteClient/Constants/Constants.swift

```diff
@@ -102,6 +102,7 @@ public struct Constants {
 
         public static let autocompleteSection = "autocomplete_section"
         public static let searchTerm = "term"
+        public static let searchTermRegex = ".*[A-Za-z0-9].*"
```

Comment:
for the story, (https://app.clubhouse.io/constructorio/story/4638/as-an-engineer-if-a-string-without-characters-is-passed-in-for-the-search-term-then-i-should-ignore-it-so-it-doesn-t-generate)  we really only care about not using nulls... the regex is prolly too specific here